packages/example-completeness/src/AttributeNormalizer.js

```diff
@@ -1,69 +1,52 @@
-import _ from 'lodash';
-import jp from 'jsonpath';
+const _ = require('lodash');
+const jp = require('jsonpath');
```

Comment:
We're building something for node. I am aware that this can be transpiled but I don't see value in it. Node soon will support ESM modules natively and the OSS community leans against transpiling imports in node land these days.